src/Util/Network.php

```diff
@@ -561,12 +561,12 @@ public static function isUrlAllowed($url)
 	public static function isUrlBlocked($url)
 	{
 		$host = @parse_url($url, PHP_URL_HOST);
-		if (! $host) {
-			return true;
```

Comment:
This was intentional, what is the case where we want to check a URL without a host against the domain block list?